cpp/tools/parquet/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ if(PARQUET_BUILD_EXECUTABLES)
     install(TARGETS ${TOOL} ${INSTALL_IS_OPTIONAL}
             RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
   endforeach(TOOL)
+  target_link_libraries(parquet-dump-footer ${ARROW_LIBRARIES})
```

Comment:
We don't need this because `parquet_shared`/`parquet_static` adds `arrow_shared`/`arrow_static` implicitly.

Reply:
Sent https://github.com/apache/arrow/pull/43375